pnpm-lock.yaml

```diff
@@ -8771,6 +8771,13 @@ packages:
     resolution: {integrity: sha512-37i+OaWTh9qeK4LSHPsyRC7NahnGotNuZvjLSgcPzblpHB3rrCJxAOgI5gCdKm7coonsaX1Of0ILiTcnZjbfxA==}
     dev: true
 
+  /@types/preferred-pm@3.0.0:
+    resolution: {integrity: sha512-Ub1de7EkdavsyM1KNrTb1K1QL+ISepEELELh2QWccyDcVEcyUDiGoYzzOJfonpGNwpymYXY13oRFpXQluGOC5w==}
+    deprecated: This is a stub types definition. preferred-pm provides its own type definitions, so you do not need this installed.
```

Comment:
Seems that the types package isn't needed?

Reply:
Good catch, I'll update this when possible!